packages/type-definitions/src/types_18.ts

```diff
@@ -6,15 +6,16 @@
  */
 
 import type { RegistryTypes } from '@polkadot/types/types'
+import { subtype } from './subtype.js'
 import { types17 } from './types_17.js'
 
-// Remove old DID types
-delete types17.DidCreationOperation
-delete types17.DidUpdateOperation
-delete types17.DidDeletionOperation
```

Comment:
not saying we have to do that, but an alternative to the subtype function would be object destructuring with remainder: 
```typescript
const {DidCreationOperation, DidUpdateOperation, DidDeletionOperation, ...typesWeKeep} = types17

export const types18: RegistryTypes = {
   ...typesWeKeep,
}
```

Reply:
We get an unused warning for `DidCreationOperation, DidUpdateOperation, DidDeletionOperation` in that case 🤓

Reply:
ignore it!